promoter/accounts.go

```diff
@@ -0,0 +1,56 @@
+package promoter
```

Comment:
nit: I would typically spin this off in its own package.

Reply:
Why is that? Seems a bit overkill to move it to a different folder. I'd rather just leave this open for a future refactor where the accounts repo itself exposes a library to use like skyd.

Reply:
Yeah, it might be an overkill at this stage.